test/helpers/balanceDiff.js

```diff
@@ -0,0 +1,10 @@
+async function balanceDifference (account, method) {
+    let balanceBefore = web3.eth.getBalance(account);
+    await method(account);
```

Comment:
This is probably a typo, since you don't use it this way in your test, but `method `shouldn't receive any parameters.